seqr/views/apis/anvil_workspace_api.py

```diff
@@ -240,3 +268,89 @@ def _send_load_data_slack_msg(project, ids_path, data_path, sample_type, user):
     )
 
     safe_post_to_slack(SEQR_SLACK_ANVIL_DATA_LOADING_CHANNEL, message_content)
+
+
+def generate_AnVIL_variables(project, data_path, sample_type):
```

Comment:
for all of these helper functions, since they are only used in this file its best to mark them as private by starting all of the function names with an underscore (`_generate_AnVIL_variables` instead of ` generate_AnVIL_variables`)